apollo-router/src/spec/field_type.rs

```diff
@@ -112,7 +112,18 @@ fn validate_input_value(
     match type_name.as_str() {
         "String" => return from_bool(value.is_string()),
         // Spec: https://spec.graphql.org/June2018/#sec-Int
-        "Int" => return from_bool(value.is_valid_int_input()),
+        "Int" => {
+            let valid = value.is_valid_int_input();
+
+            if !valid {
+                tracing::warn!(
+                    "Input INT '{}' is larger than 32-bits and is not GraphQL spec-compliant.",
+                    value.to_string()
+                )
+            }
+
+            return from_bool(valid);
```

Comment:
I'm confused about how this will  logging will work. Since `valid` comes from the `is_valid_int_input` function that you changed to return `true` as long as the input value is an `i64`.

So `!valid`here doesn't mean > 32 bits does it?